File: 217-contains-duplicate/contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        dup = set()
        for n in nums:
            if n in dup:
                return True
            dup.add(n)
        return False

File: 217-contains-duplicate/contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        int n = nums.length;
        HashSet<Integer> set = new HashSet<>();
        for(int val:nums){
            if(!set.contains(val)){
                set.add(val);
            }
            else{
                return true;
            }
        }
        return false;
    }
}